### ISSUE TITLE
[Request] add I18N tutorial and idea about missing topics

### ISSUE THREAD

**Issue by Just-some-one** [java/devjava-content#102]
it would be good to add some tutorial about i18n and how to write multinational app in java

extending that one 
maybe for all missing topic you can add the topic and add one paragraph about it and link to the old java tutorial equivalence for example the [i18n tutorial](https://docs.oracle.com/javase/tutorial/i18n/index.html)  and add that old link on that paragraph and mention that it based on jdk and you will or will not update it (in case if applicable ) 

so the developer get the full sight and know there other topic that java has or cover

for example creating GUI section and under it 2 section
1) swing with link to the old java tutorial and if it will get update later or not (based on api change )
2) javafx with link to the old java tutorial and also the new tutorial that already on dev.java till it all migrated and remove the old one 

the old tutorial can be used as base resource (as you already did with some of the current tutorial) and you update only the parts needed to save you some time 

same thing for thread and other resource 

another thing what about add roadmap for the knowledge required for java developer despite that knowledge will get tutorial or not
and if not is it possible to get question about resources for it if not also that not problem but at least it will give the developer overview about the path 

for example dev will need git and linux algorithm data structure jakartaee spring testing tool build tool and so on
it would be nice if that road map ordered in good way so for example if someone learn git then start learn java it fine they do not depend on each other but in my humble opinion learn java first then git so you can use the code knowledge to put that code on git  
someone would say we can train by text file that ok but push and commit real work even if it sample app would make it more sense 

sorry for long post and have a nice day everyone :)

**Comment by danthe1st** [comment]
For roadmaps, I think that's a bit difficult as it varies from person to person. Some want to learn backend dev with Spring, others want to do Android devs, maybe some people just want to do Minecraft modding etc so I don't think there's much of a point in creating a roadmap containing many non-Java SE things.

However, I see no issue with having a roadmap and/or graph connecting articles, how they are dependent on each other and a suggested learning path for Java (and Java alone), though this might require a few more articles on topics for that being written beforehand.

Also, if something like this is planned, it may or may not be useful to add metadata about prerequisites to each article (i.e. a list of article IDs aboug topics that people should know before).

**Comment by Just-some-one** [comment]
i agree there many roadmap but the issue that when someone new to programing and want to learn it and today there many resources and paths new person could be lost 

so it could be like hint for what java capable of like if someone interested in desktop he/she can take that path for android go that path enterprise go that path and so on despite if the resources for learning it will be on that site or any other place even without mention that other site that teach it if that issue

many would go to the official page (despite what official mean for each topic) for whatever thing they want to learn and see if they offer applicable resource (applicable based on learner point of view could be financial based - technicality level - type of material and so on)

roadmap for topic is good idea can achieved  by ordering article in some way or graph as you mentioned

but it's ok to drop the roadmap if that not beneficial to learner 

thanks for your comment and have a nice day :)
